config/application.yml.default

```diff
@@ -419,6 +419,8 @@ development:
   state_tracking_enabled: true
   telephony_adapter: test
   use_dashboard_service_providers: true
+  usps_eipp_sponsor_id: '222222222222222'
+  usps_ipp_sponsor_id: '111111111111111'
   usps_ipp_transliteration_enabled: true
```

Comment:
I wondered why these did not resolve once I merged to main. I realized I did not alpha order.  Be sure to keep your version.